doc/content/lfr/heterogeneous_single_assembly_3D/Griffin_standalone_LFR.md

```diff
@@ -0,0 +1,277 @@
+
+The active fuel region in the middle is located between upper and lower thermal insulators, gas planum, bundle grid and pins plug, and inlet and outlet wrappers. Bundle grid and pins plug, inlet and outlet wrappers, and lower core plate are radially homogenized while radially heterogeneous fine-meshes were used in the other five axial regions. The annular fuel shape in [annularfuel] is explicitly modeled for the active fuel region and materials inside the cladding are radially homogenized for the four axial regions of thermal insulators and gas planum.
+
+!table id=lfrkeyparam caption=Key parameters of Lead-Cooled Fast Reactor (LFR) model
+| Parameter | Value  | Unit  |
+| :- | :- | :- |
+| Thermal power | $950$ | $MW$  |
+| Active core height | $105$  | $cm$ |
+| Number of fuel pins per assembly | $127$ | $-$ |
+| Gap thickness | $0.175$ | $mm$ |
+| Duct thickness | $3.5$ | $mm$ |
+| Fuel pin diameter | $10.7$ | $mm$ |
+| Fuel pin pitch | $13.3$ | $mm$ |
+| P/D | $1.24$ | $-$ |
+| Cladding thickness | $0.9$ | $mm$ |
+| Fuel pellet outer diameter | $8.55$ | $mm$ |
+| Fuel pellet inner diameter | $4.0$ | $mm$ |
+| Reynolds number | $51,000$ | $-$ |
+| Prandtl number | $0.0136$ | $-$ |
+| Peclet number | $705.21$ | $-$ |
+
+!media lfr/LFRgeometry.jpg
+       style=width:80%
+       id=lfrgeom
+       caption=Axial and radial geometry of the LFR single assembly model
+
+!media lfr/annularfuel.png
+       style=width:30%
+       id=annularfuel
+       caption=LFR annular fuel pin geometry
+
+## Cross Section Generation using MC2-3
+
+It is worth explaining the procedure to generate cross sections using MC2-3 for this model. The axial leakage effect needs to be considered using the MC2-3/TWODANT two-step procedure for the generation of both homogeneous cross sections for non-fuel axial regions and heterogeneous region-wise cross sections for the fuel axial region. At this point of time, the MC2-3/TWODANT procedure is not fully available in Griffin. Once the feature is available in Griffin, the description below will be updated.
```

Comment:
`the MC2-3/TWODANT procedure is not fully available in Griffin` => `the MC2-3/TWODANT procedure is not fully available in Griffin due to a licensing issue with TWODANT`. Also please mention MC2-3 is the cross section generation module in Griffin.